pkgs/tools/package-management/appimagekit/default.nix

```diff
@@ -72,9 +72,13 @@ in stdenv.mkDerivation rec {
   buildInputs = [
     glib zlib cairo openssl fuse
     xz inotify-tools libarchive
-    squashfsTools
+    squashfsTools makeWrapper
   ];
 
+  postPatch = ''
+    substituteInPlace src/appimagetool.c --replace "/usr/bin/file" "${file}/bin/file"
```

Comment:
You don't need this if you have wrapper?

Reply:
I believe so, since it is pointing to a hardcoded path?

Reply:
In that case you may not need the wrapper for file below. I guess, since you already need that for gnupg, it's fine to leave it as is.